apollo-router/src/plugins/snapshots/apollo_router__plugins__expose_query_plan__tests__it_expose_query_plan_without_executing.snap

```diff
@@ -0,0 +1,173 @@
+---
+source: apollo-router/src/plugins/expose_query_plan.rs
+expression: "serde_json::to_value(response).unwrap()"
+---
+{
+  "extensions": {
+    "apolloQueryPlan": {
+      "object": {
+        "kind": "QueryPlan",
+        "node": {
+          "kind": "Sequence",
+          "nodes": [
+            {
+              "kind": "Fetch",
+              "serviceName": "products",
+              "variableUsages": [
+                "first"
+              ],
+              "operation": "query TopProducts__products__0($first:Int){topProducts(first:$first){__typename upc name}}",
+              "operationName": "TopProducts__products__0",
+              "operationKind": "query",
+              "id": null,
+              "inputRewrites": null,
+              "outputRewrites": null,
+              "contextRewrites": null,
+              "schemaAwareHash": "7245d488e97c3b2ac9f5fa4dd4660940b94ad81af070013305b2c0f76337b2f9",
+              "authorization": {
+                "is_authenticated": false,
+                "scopes": [],
+                "policies": []
+              }
+            },
+            {
+              "kind": "Flatten",
+              "path": [
+                "topProducts",
+                "@"
+              ],
+              "node": {
+                "kind": "Fetch",
+                "serviceName": "reviews",
+                "requires": [
+                  {
+                    "kind": "InlineFragment",
+                    "typeCondition": "Product",
+                    "selections": [
+                      {
+                        "kind": "Field",
+                        "name": "__typename"
+                      },
+                      {
+                        "kind": "Field",
+                        "name": "upc"
+                      }
+                    ]
+                  }
+                ],
+                "variableUsages": [],
+                "operation": "query TopProducts__reviews__1($representations:[_Any!]!){_entities(representations:$representations){...on Product{reviews{id product{__typename upc}author{__typename id}}}}}",
+                "operationName": "TopProducts__reviews__1",
+                "operationKind": "query",
+                "id": null,
+                "inputRewrites": null,
+                "outputRewrites": null,
+                "contextRewrites": null,
+                "schemaAwareHash": "6e0b4156706ea0cf924500cfdc99dd44b9f0ed07e2d3f888d4aff156e6a33238",
+                "authorization": {
+                  "is_authenticated": false,
+                  "scopes": [],
+                  "policies": []
+                }
```

Comment:
This is all just used by Explorer, right? Do we have a representation of the query plan in JSON format that doesn't include all this additional information that Explorer doesn't need?